website/docs/getting-started/setup/installation-guides/aws-ecs-with-fargate.md

```diff
@@ -0,0 +1,218 @@
+# AWS ECS with Fargate
+
+## Prerequisites
+
```

Comment:
Can we not subheadings for pre-requisites and instead directly number them as 

1. [Register with Amazon Web Services](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/) - You may skip this step if you already have an AWS account.
2. [Create a security group](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/working-with-security-groups.html#adding-security-group-rule) - You can skip this step if you already have an existing security group with ports 80 and 443. Appsmith requires ports 80 and 443 for HTTP access. To edit the **Inbound Rules**  and make ports 80 and 443 accessible from anywhere, see [Add rules to a security group](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/working-with-security-groups.html#adding-security-group-rule)
...... and so on

Reply:
For pre-requisities that need further explanation like **Create an EFS filesystem**, you can do it as bullets under the main point.